_docs/demo/setup.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: page
+title: Setup
+permalink: /docs/demo/setup/
+category: Demo
+order: 1
+---
+
+While Amalgam8 supports multi-tenancy, for the sake of simplicity, this
+demo will setup Amalgam8 in single-tenant mode.
+
+## Download
+
+Download and unzip the latest release of Amalgam8 demos from the
+[Amalgam8 Releases](https://www.github.com/amalgam8/amalgam8/releases)
```

Comment:
A note about this: the release is called "a8demos-v0.3.1.zip" but the folder inside is called "examples"